src/Solver/Interoperability.cpp

```diff
@@ -802,11 +802,18 @@ void seissol::Interoperability::initializeIO(
 
   constexpr auto numberOfQuantities = tensor::Q::Shape[ sizeof(tensor::Q::Shape) / sizeof(tensor::Q::Shape[0]) - 1];
 
+  // record the clustering info i.e., distribution of elements within an LTS tree
+  const std::vector<Element>& MeshElements = seissol::SeisSol::main.meshReader().getElements();
+  std::vector<unsigned> ClusteringFieldMap(MeshElements.size());
+  for (const auto& Element: MeshElements) {
+    ClusteringFieldMap[Element.localId] = m_ltsLut.cluster(Element.localId);
```

Comment:
Are you sure that this tracks the global cluster id and not the local one?  (They can differ!)
It probably tracks the global one but I'm not sure.

Reply:
I've fixed it. 

Regarding tabs and white spaces. I can see that it is getting messy on github. I set my IDE to use spaces instead of tabs. Which setting do you use?

Reply:
Think much of the C++ code uses two spaces. But in total it's a mess.